Source/WebCore/accessibility/cocoa/AXTextMarkerCocoa.mm

```diff
@@ -91,7 +92,40 @@
         , m_end.platformData().autorelease()
     ));
 }
-#endif // PLATFORM(MAC)
+
+#elif PLATFORM(IOS_FAMILY)
+
+AXTextMarkerRange::AXTextMarkerRange(NSArray *markers)
+{
+    if (markers.count != 2)
+        return;
+
+    WebAccessibilityTextMarker *start = [markers objectAtIndex:0];
+    WebAccessibilityTextMarker *end = [markers objectAtIndex:1];
+    if (![start isKindOfClass:[WebAccessibilityTextMarker class]] || ![end isKindOfClass:[WebAccessibilityTextMarker class]])
+        return;
+
+    m_start = { [start textMarkerData ] };
+    m_end = { [end textMarkerData] };
+}
+
+NSArray *AXTextMarkerRange::platformData() const
+{
+    if (!*this)
+        return nil;
+
+    RefPtr object = m_start.object();
+    ASSERT(object); // Since *this is not null.
+    auto* cache = object->axObjectCache();
```

Comment:
Should we make the cache a CheckedPtr rather than a raw pointer?

Reply:
Decided to leave it as is because the only use is to copy the pointer into WebAccessibilityTextMarker. I'm following up with another change that actually removes the pointer from WebAccessibilityTextMarker so this becomes unnecessary.

Reply:
Sounds good!